examples/io/elekta_average.py

```diff
@@ -0,0 +1,62 @@
+"""
+================================
+Getting averaging info from fiff
+================================
+
+Get averaging information defined in Elekta Vectorview/TRIUX DACQ (data
+acquisition). Extract and average epochs accordingly.
+"""
+# Author: Jussi Nurminen (jnu@iki.fi)
+#
+# License: BSD (3-clause)
+
+
+import mne
+import os
+from mne.datasets import testing
+from mne.event import ElektaAverager
+
+elekta_base_dir = os.path.join(testing.data_path(), 'misc')
+fname_raw_elekta = os.path.join(elekta_base_dir, 'test_elekta_3ch_raw.fif')
+
+print(__doc__)
+
+###############################################################################
+# Read raw file and create averager instance
+raw = mne.io.read_raw_fif(fname_raw_elekta)
+eav = ElektaAverager(raw.info)
+
+###############################################################################
+# Check DACQ defined averaging categories and other info
+print(eav)
+
+###############################################################################
+# Extract epochs corresponding to a category
+cond = eav.get_condition(raw, 'Test event 3')[0]  # always returns a list
```

Comment:
see API of read_evokeds

if you specify a condition name you don't get a list. You should get a list only if you don't specify condition names.